static/styles/hotspots.css

```diff
@@ -0,0 +1,183 @@
+        opacity: 0.8;
+    }
+
+    100% {
+        -webkit-transform: scale(2.2, 2.2);
+        opacity: 0.0;
+    }
+}
+
+
+/* popover */
+.hotspot .hotspot-popover-wrapper {
+    position: relative;
+    opacity: 0;
+    cursor: default;
+}
+
+.hotspot .hotspot-popover {
+    display: none;
+    position: absolute;
+    width: 250px;
+    text-align: left;
+    box-shadow: 0 5px 10px rgba(0, 0, 0, 0.2);
+    border: 1px solid hsl(0, 0%, 80%);
+    border-radius: 4px;
+}
+
+.hotspot .hotspot-popover-top {
+    padding: 0 15px;
+    border-top-left-radius: 4px;
+    border-top-right-radius: 4px;
+    background-color: hsl(215, 47%, 50%);
+}
+
+.hotspot .hotspot-title {
+    margin: 0;
+    font-size: 1.15em;
+    font-weight: 600;
+    color: white;
+}
+
+.hotspot .hotspot-popover-bottom {
+    background-color: white;
+    padding: 15px;
+    height: 130px;
+    border-bottom-left-radius: 4px;
+    border-bottom-right-radius: 4px;
+}
+
+.hotspot .hotspot-confirm {
+    position: absolute;
+    bottom: 18px;
+    right: 18px;
+    border: none;
+    background-color: hsl(215, 47%, 50%);
+    color: white;
+    font-size: 1.05em;
+    font-weight: 500;
+    width: 70px;
+    height: 28px;
+    border-radius: 4px;
+}
+
+/* arrows */
+.hotspot-popover:after,
+.hotspot-popover:before {
+    border: solid transparent;
+    content: '';
+    height: 0;
+    width: 0;
+    position: absolute;
+    pointer-events: none;
+}
+
+.hotspot-popover:after {
+    border-width: 12px;
+}
+.hotspot-popover:before {
+    border-width: 13px;
+}
+
+.hotspot-popover.arrow-top:before,
+.hotspot-popover.arrow-top:after {
+    bottom: 100%;
+    right: 50%;
+}
+.hotspot-popover.arrow-top:after {
+    border-bottom-color: hsl(215, 47%, 50%);
+    margin-right: -12px;
+}
+.hotspot-popover.arrow-top:before {
+    border-bottom-color: hsl(0, 0%, 80%);
+    margin-right: -13px;
+}
+
+.hotspot-popover.arrow-left:before,
+.hotspot-popover.arrow-left:after {
+    right: 100%;
+    top: 50%;
+}
+.hotspot-popover.arrow-left:after {
+    border-right-color: #fff;
+    margin-top: -12px;
+}
+.hotspot-popover.arrow-left:before {
+    border-right-color: hsl(0, 0%, 80%);
+    margin-top: -13px;
+}
+
+.hotspot-popover.arrow-bottom:before,
+.hotspot-popover.arrow-bottom:after {
+    top: 100%;
+    right: 50%;
+}
+.hotspot-popover.arrow-bottom:after {
+    border-top-color: #fff;
+    margin-right: -12px;
+}
+.hotspot-popover.arrow-bottom:before {
+    border-top-color: hsl(0, 0%, 80%);
+    margin-right: -13px;
+}
+
+.hotspot-popover.arrow-right:before,
+.hotspot-popover.arrow-right:after {
+    left: 100%;
+    top: 50%;
+}
+.hotspot-popover.arrow-right:after {
+    border-left-color: #fff;
+    margin-top: -12px;
+}
+.hotspot-popover.arrow-right:before {
+    border-left-color: hsl(0, 0%, 80%);
+    margin-top: -13px;
+}
+
+/* individual hotspots */
+#hotspot_click_to_reply {
+    z-index: 2;
+}
+
+#hotspot_stream_settings {
+    z-index: 2;
+}
```

Comment:
Should a high `z-index` just be part of the general hotspots CSS, rathern than being custom for each hotspot?